tensorflow/core/kernels/reshape_op.cc

```diff
@@ -86,7 +86,7 @@ REGISTER_KERNEL_BUILDER(Name("Reshape")
 #undef REGISTER_SYCL_KERNEL
 #endif  // TENSORFLOW_USE_SYCL
 
-#if GOOGLE_CUDA
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
```

Comment:
nit. also update #endif?

Reply:
Sure, no prob.  Will update this and my upcoming PRs too.  ;-)